app/views/mentor_terms/_fields.html.erb

```diff
@@ -1,7 +1,10 @@
 <%= f.inputs "Options" do  %>
 	<%= f.input :lead, :label => Customer.lead_Label %>
 	<%= f.input :volunteer if Customer.link_to_uw? %>
-	<%= f.input :driver %>
+	<%= f.input :driver %> 
+	<%= f.input :steering, :label => "Steering" %>
+	<%= f.input :ccra, :label => "CCRA" %>
+	<%= f.input :planning, :label => "Planning" %>
```

Comment:
@aayush19saxena Do you think we should use fully dynamic tags instead here? These are so specific to the program and if you ever want to change them it will require updating code.